docs/README.md

```diff
@@ -5,12 +5,13 @@ description: All Netlify CLI commands
 
 # Netlify CLI Command List
 
-Welcome to the Netlify CLI! This site provides online access to all help strings in the Netlify CLI. For a more in-depth guide, please see our [Getting Started](https://docs.netlify.com/cli/get-started/) guide on our main docs site.
+Welcome to the Netlify CLI! This site provides online access to all help strings in the Netlify CLI. For a more in-depth
+guide, please see our [Getting Started](https://docs.netlify.com/cli/get-started/) guide on our main docs site.
 
-If you have questions, ideas, or would like to contribute, check out the [repository on GitHub](https://github.com/netlify/cli/).
+If you have questions, ideas, or would like to contribute, check out the
+[repository on GitHub](https://github.com/netlify/cli/).
 
-**Before you begin**
-Make sure you have [Node.js](https://nodejs.org/en/download/) version 12.20.0, 14.14.0, 16.0.0, or later.
+**Before you begin** Make sure you have [Node.js](https://nodejs.org/en/download/) version 16.0.0, or later.
```

Comment:
Also removed 14 here?

Reply:
Yea this was intentional as we want folks to install at least the current LTS 16 :) 14 will be EOL in March 2023 so there is no reason to keep install instructions on instructing for old versions.

always go with LTS.

Regarding the 14.18.x version I was thinking the same but as this is a rather new version I doubt that all folks have it installed. I would love to do that but 14.16 is already pretty old, released on 2021-02-23. 

As said this should be a minimum required version. Furthermore this version was communicated in the forums post: https://answers.netlify.com/t/netlify-cli-dropping-support-for-node-js-version-12/75130

Reply:
Yeah I guess we can wait for that for one more year :) Maybe we can change the workflow to use 14.16 so that in case we update dependencies that use `node:` the workflow would catch that.